src/sentry/web/frontend/base.py

```diff
@@ -407,28 +407,31 @@ def handle_disabled_member(self, organization: Organization) -> HttpResponse:
 
 class OrganizationView(BaseView):
     """
-    Any view acting on behalf of an organization should inherit from this base.
-
-    The 'organization' keyword argument is automatically injected into the
-    resulting dispatch.
+    A deprecated view used by endpoints that act on behalf of an organization.
```

Comment:
Future PRs will begin moving existing endpoints to one of either `RegionSiloOrganizationView` or `ControlSiloOrganizationView`.  This will take some time since it also means splitting the `ReactPageView` along similar lines.  That said, this existing class remains and works as it had before, and is fine to operate in both modes (as well as SaaS).  The main advantage of splitting it though is clarity, esp of types.